cli/src/lib.rs

```diff
@@ -181,6 +181,8 @@ pub enum Command {
         #[clap(subcommand)]
         subcmd: IdlCommand,
     },
+    /// Remove generated artifacts.
```

Comment:
also say what is *not* being deleted

Reply:
Changed to `Remove all artifacts from the target directory except program keypairs`